src/main/java/com/example/demo/service/PersonService.java

```diff
@@ -22,6 +22,7 @@ public int addPerson(Person person) {
         return personDao.insertPerson(person);
     }
 
+    //get toate persoanele
```

Comment:
The comment should be in English. Please modify

Reply:
Done